src/index.d.ts

```diff
@@ -55,6 +71,8 @@ interface Slots {
 }
 /**
  * Meta.
+ * 
+ * @deprecated See https://github.com/storybookjs/addon-svelte-csf/issues/135
```

Comment:
It might be more helpful to just link to the updated readme?
```suggestion
 * @deprecated Use `export const meta`. See https://github.com/storybookjs/addon-svelte-csf for an example
```